_scss/_night-mode.scss

```diff
@@ -105,105 +75,55 @@ body.night {
         background: #2b4252;
     }
     pre {
-        display: block;
-        padding: 9.5px;
-        margin: 0 0 10px;
-        font-size: 13px;
-        line-height: 1.42857143;
         color: #f3f3f3;
-        word-break: break-all;
-        word-wrap: break-word;
         background-color: #0a121b !important;
         border: 1px solid $black;
-        border-radius: 4px;
     }
     code {
-        padding: 3px 7px;
         color: #c1f1f0 !important;
-        border-radius: 2px;
         background: #0a121b !important;
     }
-    pre>code {
-        padding: 0;
-    }
     .nav-secondary .dropdown-btn {
         color: #d0e8ff;
     }
     .outline-btn {
         background: #101c29;
-        border: 1px solid;
-        border-color: #37404a;
-        text-decoration: none;
-        margin: 0;
+        border: 1px solid #37404a;
     }
     a.button.outline-btn {
         color: #a7e2ff;
-        float: none;
-        margin-bottom: 30px;
-        display: inline-block;
     }
     a.glossary,
         .toc-nav i.fa{
             color: $body-text-night;
         }
-    .component img, .component-full-icon img {
+    .component img {
         opacity: 1;
         -webkit-filter: grayscale(100%);
         filter: grayscale(100%);
     }
-    .hljs {
-        color: #f5f5f5!important;
-    }
-    .hljs-tag,
-    .hljs-name,
-    .hljs-attribute {
-        color: #e3e3f2!important;
-        font-weight: normal;
-    }
-    .hljs-keyword,
-    .hljs-selector-tag,
-    .hljs-subst {
-        color: #f5f5f5!important;
-    }
-    div#autocompleteResults {
+    #autocompleteResults {
         background: $bg-search-results-night;
         border: 1px solid $black;
-        color: $white;
-        padding: 20px 0 15px 0;
-        margin: 10px 0 0 0;
-        position: absolute;
-        width: 800px;
-    }
-    ul.autocompleteList {
-        list-style: none;
-    }
-    .autocompleteList li {
-        color: #b4c3d2;
-        padding: 5px 0 7px 0;
+        //color: $white;
     }
-    div#autocompleteResults span {
-        color: #b7a4de;
-    }
-    .autocompleteTitle {
-        font-weight: bold;
-    }
-    .autocompleteUrl,
-    .autocompleteKeywords,
-    .autocompleteDescription {
-        line-height: normal;
+    .autocompleteResult {
+        &.selected {
+            background-color: #0f161e;
```

Comment:
This looks a bit dark for the selected search result:

![navigating-search-result-dark](https://user-images.githubusercontent.com/207759/95713831-7b9d4000-0c67-11eb-9896-1d519bb0c7f7.gif)


Maybe a bit lighter. 🤔 

```suggestion
            background-color: #1f262e;
```

Reply:
Yes, I also found it too dark; this was the color that was "meant" to be picked so I kept that color, but let me tweak it a bit to my own taste 😅